packages/terra-popup/src/Popup.jsx

```diff
@@ -40,17 +56,19 @@ const propTypes = {
    */
   classNameOverlay: PropTypes.string,
   /**
-   * Attachment point for the popup, this will be mirrored to the target.
+   * Attachment point for the popup, this will be mirrored to the target. Options include: 'top left',
+   * 'top center', 'top right', 'middle left', 'middle center', 'middle right', 'bottom left',
+   * 'bottom center', or 'bottom right'.
    */
-  contentAttachment: PropTypes.oneOf(TetherComponent.attachmentPositions),
+  contentAttachment: PropTypes.oneOf(ATTACHMENT_POSITIONS),
```

Comment:
Should we make this a shape to match hookshot's prop?

```
  contentAttachment: PropTypes.shape({
    horizontal: PropTypes.oneOf(Hookshot.HORIZONTAL_ATTACHMENTS),
    vertical: PropTypes.oneOf(Hookshot.VERTICAL_ATTACHMENTS),
  }),
```

Reply:
It wouldn't be bad, but I know we were hoping to keep the API passive. If we plan on doing a 2.0 and are fine with breaking passivity we could.

Reply:
After discussion for the time being we'll leave it as a minor version bump.